ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -84,43 +82,15 @@ tests:
           memory: 200Mi
     - chain: openshift-upgrade-qe-test-eus
     workflow: cucushift-installer-rehearse-azure-ipi-workers-rhel8
-- as: baremetalds-ipi-ovn-ipv4-fips-f28
-  cron: 22 2 5 * *
-  steps:
-    cluster_profile: equinix-ocp-metal-qe
-    env:
-      DEVSCRIPTS_CONFIG: |
-        FIPS_MODE=true
-        IP_STACK=v4
-        NETWORK_TYPE=OVNKubernetes
-      FILTERS_ADDITIONAL: ~TELCO&
-      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.12 and @baremetal-ipi'
-      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.12 and @baremetal-ipi'
-    test:
-    - as: set-upgrade-releases
-      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/upgrade-edge
-      dependencies:
-      - env: RELEASE_IMAGE_INTERMEDIATE
-        name: release:intermediate
-      - env: RELEASE_IMAGE_TARGET
-        name: release:target
-      from: cli
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
-    - chain: openshift-upgrade-qe-test-eus
-    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
-- as: gcp-ipi-oidc-rt-fips-f28
-  cron: 23 3 3 * *
+- as: gcp-upi-private-xpn-sdn-f14
+  cron: 24 6 8,24 * *
   steps:
     cluster_profile: gcp-qe
     env:
+      ADDITIONAL_ENABLED_CAPABILITIES: baremetal marketplace openshift-samples CSISnapshot
```

Comment:
If we are testing the capacity on purpose, we need to update job name accordingly to append `basecap-none-additionalcaps` keyword there.

Reply:
Then I prefer not test caps.

Reply:
I am okay with either. If we do not the capacity on this job, we need to drop the setting here.

Reply:
Removed cap settings, right now we don't have capability cases in 4.12 -> 4.14 eus upgrade. cc @shellyyang1989 @jianlinliu @jiajliu